### ISSUE TITLE
Wrong power supply dimension

### ISSUE THREAD

**Issue by LemarinelNet** [arbalet-project/arbadoc#5]
Hi,

I've made a small calculation : 
Each WS2812B module contains 3 LED, and each led can consume up to 20 mA.
This leads to 60mA per LED.
So, for a table containing 150 LED, if each led is set at full power white, we need 150 x 0.06A = 9 A !

I think that asking for a 5V / 5A power supply could cause to strange behavior ;)

It should be more reasonable to ask for a 10 A PSU.

PS : Boujou de normandie !

**Comment by AurelienC** [comment]
Hello,

Good remark.

I don't have find much documentation, but I find 50mA at 5V for the full brightness of R, G and B LEDs.
This is about 7.5 A for 150 LEDs.

Indeed, a very powerful PSU must be a good thing, but not necessary with a "normal" use of the table. A good 5A power supply can provide most power in a short period of time and each LED has a tank capacitor.

**Comment by LemarinelNet** [comment]
Yes, that's right, for normal use, and if we don't use as a giant torch, 5A is certainly enough :)
Perhaps we should warn in the doc that 5A is sufficient for normal use, but at full power it will not be enough...

Also, what about a french documentation ?

**Comment by ymollard** [comment]
It probably depends on the LED strips. I've made some measurements on my strips from different manufacturers, powered with a lab power supply, I actually never needed more than 3.5A even at full brightness. I have the feeling that the theoretical 50mA per LED is a bit overkill especially when multiplied by 150. Strong power supplies are bigger, this is maybe not something you want close to your coffee table.

>  Also, what about a french documentation ?

I do accept pull requests on that repository :)

**Comment by LemarinelNet** [comment]
I can't remember where I did saw those 20mA per LED, but nothing in the datasheet about IF current !!

It might be a general rule about led : when I did my high school, we did always took 20mA for a LED current... nevermind....

If I had some free time, I would translate the pages... perhaps someday ;)

**Comment by LemarinelNet** [comment]
How should I proceed to translate the docs ? Should I suffix each file with "_fr". i.e. : assembly_guidelines_arbalet_lava_fr.odt ?
Or should I create a different subtree ?

**Comment by ymollard** [comment]
If you follow the advice given by LED strips manufacturers they also suggest 7.5 amps or so.

Adding `_fr` is fine. You'll need the [`Oxygen`](https://fonts.google.com/specimen/Oxygen?selection.family=Oxygen) font.

**Comment by LemarinelNet** [comment]
Great, I'll start translating the docs !
I'm not very used to Github. 
Should I first fork the projects to be able to pull request after ?

(PS : So silly to discuss in english since we are both french !!)

**Comment by ymollard** [comment]
Great! Yes, fork the project, then clone your own copy on your local disk (on Linux `git clone https://github.com/LemarinelNet/arbadoc.git`), then create the new file, then `git add the_new_file.odt`, then `git commit`, type a description of your work and save the result, you can do several commits and at the end `git push` to save your work online. The pull request is made afterwards from github.

**Comment by LemarinelNet** [comment]
Ok ! I'll work on it as soon I'll have some free time :)
